spec/support/helpers/api_helpers.rb

```diff
@@ -30,6 +30,10 @@ def parsed_response
       Oj.load(response.body)
     end
 
+    def response_ids
+      parsed_response["data"].map { |statement| statement["id"] }
```

Comment:
this is changing since Leandro's pr went in with `ecf_id`

Reply:
@cwrw `id` here is correct. Leandros change to the serializer should return `ecf_id` as `id` in the JSON response.

@pmanrubia could you rename the variable `statement` to something general like `data`, as this helper is not specific to statement.

Reply:
ah sorry your right!